Dockerfile.builder

```
@@ -1,4 +1,4 @@
-FROM golang:1.26.5
+FROM golang:1.26.6
```

RuboCop (1.88.2)

[fatal] no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)